ci/test-conformance-gke.sh

```diff
@@ -140,7 +140,7 @@ fi
 function setup_gke() {
 
     if [[ -z ${K8S_VERSION+x} ]]; then
-        K8S_VERSION="1.15.12-gke.3"
+        K8S_VERSION=$(${GCLOUD_PATH} container get-server-config --zone ${GKE_ZONE} | awk '/validMasterVersions/{getline;print}' | cut -c3- )
```

Comment:
if we just omit the `--cluster-version` command-line option in that case, wouldn't that work as well?

Reply:
It will use the default version rather than the highest supported version. I remember having some headaches around that since sonobouy had a minimum k8s version requirement. Might be ok now though.